adapter/noop2/noop.go

```diff
@@ -82,37 +84,41 @@ func (builder) ConfigureQuotaHandler(map[string]*quota.Type) error {
 
 ////////////////// Runtime Methods //////////////////////////
 
-var cacheInfo = adapter.CacheabilityInfo{
+var checkResult = adapter.CheckResult{
+	Status:        rpc.Status{Code: int32(rpc.OK)},
 	ValidDuration: 1000000000 * time.Second,
 	ValidUseCount: 1000000000,
 }
+var reportResult = adapter.ReportResult{
```

Comment:
Well, there just isn't any more state to return for Report.

Check needs has 3 total states:
  - Thumbs Up (status == OK, error == nil)
  - Thumbs Down (status == <some error>, error == nil)
  - Unknown (status == nil, error  == <RPC error string>)

Report on the other hand does not have 3 states to return, it has two:
  - Data was reported (error == nil)
  - Some data was not reported (error != nil)